package/MDAnalysis/core/universe.py

```diff
@@ -170,62 +229,65 @@ class Universe(object):
 
     Parameters
     ----------
-    topology : str, Topology object or stream
-        A CHARMM/XPLOR PSF topology file, PDB file or Gromacs GRO file; used to
-        define the list of atoms. If the file includes bond information,
-        partial charges, atom masses, ... then these data will be available to
-        MDAnalysis. A "structure" file (PSF, PDB or GRO, in the sense of a
-        topology) is always required. Alternatively, an existing
-        :class:`MDAnalysis.core.topology.Topology` instance may also be given.
-    topology_format
+    topology : `~MDAnalysis.core.topology.Topology`, str, stream, `np.ndarray`, None
+        The topology defines the list of atoms. This can be provided 
+        as filenames, streams of filenames, numpy arrays, or None for 
+        an empty universe.
+    coordinates : str, stream, list of str, list of stream (optional)
+        Coordinates can be provided as streams of filenames either of 
+        a single frame (eg a PDB, CRD, or GRO file); a list of single 
+        frames; or a trajectory file (in CHARMM/NAMD/LAMMPS DCD, Gromacs 
+        XTC/TRR, or generic XYZ format). The coordinates have to be 
+        ordered in the same way as the list of atoms in the topology. 
+        See :ref:`Supported coordinate formats` for what can be read 
+        as coordinates.
+    topology_format: str, ``None``, default ``None``
         Provide the file format of the topology file; ``None`` guesses it from
-        the file extension [``None``] Can also pass a subclass of
+        the file extension. Can also pass a subclass of
         :class:`MDAnalysis.topology.base.TopologyReaderBase` to define a custom
         reader to be used on the topology file.
-    format
+    format: str, ``None``, default ``None``
         Provide the file format of the coordinate or trajectory file; ``None``
         guesses it from the file extension. Note that this keyword has no
         effect if a list of file names is supplied because the "chained" reader
         has to guess the file format for each individual list member.
```

Comment:
Tbh this is a minor bug, maybe can raise an issue that chainedreader should respect the format keyword.